filament/build.gradle

```diff
@@ -27,6 +27,16 @@ dependencies {
 	implementation "net.fabricmc.unpick:unpick-format-utils:$properties.unpick_version"
 	implementation "net.fabricmc:tiny-mappings-parser:$properties.tiny_mappings_parser_version"
 	implementation "net.fabricmc:tiny-remapper:$properties.tiny_remapper_version"
+	implementation "com.fasterxml.jackson.core:jackson-databind:2.13.3"
+
+	// Contains a number of useful utilities we can re-use.
+	implementation ("net.fabricmc:fabric-loom:1.0.7") {
```

Comment:
Its a bit unorthodox to pull loom in like this, but it has some useful utils that dont need to be written twice.